openupgrade_scripts/scripts/website_payment/16.0.1.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.delete_records_safely_by_xml_id(
+        env,
+        [
+            "website_payment.mail_template_donation",
+        ],
+    )
```

Comment:
I think that simply loading `noupdate_changes.xml` is enough. No need to delete this record before.

Based on: http://oca.github.io/OpenUpgrade/use_cases/noupdate_xml_entry_changed.html